pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "aws-operator"
 	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "8.8.1-dev"
+	version            = "8.8.1-xh3b4sd"
```

Comment:
Forgot to revert this. Fixed in https://github.com/giantswarm/aws-operator/pull/2683.